src/functionalTest/groovy/nebula/plugin/docker/NebulaDockerPluginTest.groovy

```diff
@@ -261,4 +261,37 @@ class NebulaDockerPluginTest extends ProjectSpec {
         project.extensions['docker'].classpath
         //TODO: how to check the list of classpath?
     }
+
+    def "enabling repoAuth and setting the username password and email on docker.registryCredentials"() {
```

Comment:
can you also add a test to ensure that flipping the switch on `dockerRepoAuth` does NOT include the credentials?